examples/hitl/basic_viewer/basic_viewer.py

```diff
@@ -46,9 +45,11 @@ def __init__(
         self._cam_transform = None
 
         self._camera_helper = CameraHelper(
-            self._sandbox_service.args, self._sandbox_service.gui_input
+            self._sandbox_service.hitl_config, self._sandbox_service.gui_input
         )
         self._episode_helper = self._sandbox_service.episode_helper
+        self._paused = False
```

Comment:
These new pause/single-step features are unrelated to the Hydra config migration. Apologies for the confusion.